pkg/security/tests/arithmetic_operations_test.go

```diff
@@ -0,0 +1,127 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build functionaltests
+// +build functionaltests
+
+package tests
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+	"github.com/DataDog/datadog-agent/pkg/security/secl/rules"
+)
+
+func TestArithmeticOperation(t *testing.T) {
```

Comment:
i like using this style of writing tests to save on duplicated code https://github.com/DataDog/datadog-agent/blob/main/pkg/security/tests/process_test.go#L2121